apps/condo/domains/notification/access/PushToken.js

```diff
@@ -0,0 +1,38 @@
+// @ts-nocheck
+/**
+ * Generated by `createschema notification.PushToken PushToken`
+ */
+
+const { throwAuthenticationError } = require('@condo/domains/common/utils/apolloErrorFormatter')
+
+async function canReadPushTokens ({ authentication: { item: user } }) {
+    // if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+    if (user.isAdmin) return {}
+    return {
+        // TODO(codegen): write canReadPushTokens logic!
+    }
+}
+
+async function canManagePushTokens ({ authentication: { item: user }, originalInput, operation, itemId }) {
+    if (user && user.deletedAt) return false
+    if (user && user.isAdmin) return true
+    if (operation === 'create') {
+        // TODO(codegen): write canManagePushTokens create logic!
+        return true
+    } else if (operation === 'update') {
+        if (!user) return throwAuthenticationError()
```

Comment:
It looks like only user can update their token

Reply:
fixed. anyone can update token if deviceId is known (almost impossible to guess, but we need brutforce protection application wide actually :) )